content/desktop/synchronized-file-sharing.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Synchronized file sharing
+description: Get started with Synchronized file sharing on Docker Desktop.
+sitemap: false
+---
+
+> **Early Access**
+>
+> Synchronized file sharing is an early-access feature. 
+>
+> If you would like to be considered for Synchronized file sharing testing, you can
+> [sign up for the early access program](https://www.docker.com/build-early-access-program/?utm_source=docs).
```

Comment:
Todo: change link

Reply:
@smnovick do you have a dedicated link for this? Or are we just going with https://www.docker.com/community/get-involved/developer-preview/